standard/attributes.md

```diff
@@ -1060,6 +1060,44 @@ Specifies that a nullable argument won’t be `null` when the method returns the
 >
 > *end example*
 
+### §enumerator-cancellation The EnumeratorCancellation attribute
+
+Specifies the parameter representing the `CancellationToken` for an asynchronous iterator (§15.15). The value of this parameter is a combination of two cancellation tokens: the token which was actually passed to the method, and the token passed to the `IAsyncEnumerable<T>.GetAsyncEnumerator(CancellationToken)` implementation on the generated iterator. The combined token will be canceled if either of the two source tokens are canceled.
```

Comment:
"will be" suggests we're requiring that the compiler observes this attribute - but later we have "The compiler can create a combined token" - let's decide whether or not a C# 8 compiler *must* implement this.

Reply:
It *is* "the method with the parameter with this annotation on it" - which I understand is surprising. (It's quite odd to pass an argument, and end up with a different value for the parameter... but that's what's happening here.)

Reply:
Reworded to try and make this more clear.